### PR TITLE
Improve and organize the routes table!

### PR DESCRIPTION
**Pull request by kofalt**
Something we've been needing for awhile, I think :slightly_smiling_face: 

Don't bother with the diff, the [result](https://github.com/scitran/core/blob/better-routes/api/api.py) is what you're after.

### CODE REVIEW

**Comment by kofalt** [comment]
> coverage decreased (-0.05%)

File: api/api.py [hunk 1]
```diff
@@ -7,210 +7,266 @@
...
+        # Search
+
+        route('/search',                     SearchHandler, h='advanced_search', m=['POST']),
+        route('/search/files',               SearchHandler, h='get_datatree',    m=['GET']),
+        route('/search/<cont_name:{cname}>', SearchHandler,                      m=['GET']),
+
+
+        # Users
+
+        route( '/users',                   UserHandler, 'get_all', m=['GET']),
```
Comment (nagem) [inline api/api.py]: Just missing the `h` here, otherwise looks good! Because python it will work without it, though 😉.

**Comment by nagem** [comment]
Merge it if you see it today, otherwise I'll merge it tonight so we can bug bash it tomorrow.

File: api/api.py (hunk 1 above)
Comment (kofalt) [inline api/api.py]: Fixed lol

**Pull request merged**